src/pages/index.js

```diff
@@ -3,20 +3,21 @@ import { SEO } from "../components/seo";
 import { Paragraph } from "../components/typography";
 import { Card, CardHeader, CardBody } from "../components/card";
 import { PageContent, Container } from "../components/layout";
-import { DataCarousel } from "../components/data-carousel";
```

Comment:
let's get rid of this entire old carousel component since, after your branch is merged, it won't be used anywhere